core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.store;
+
+import alluxio.client.file.cache.PageId;
+import alluxio.client.file.cache.PageInfo;
+import alluxio.client.file.cache.PageStore;
+import alluxio.exception.PageNotFoundException;
+import alluxio.proto.client.Cache;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Streams;
+import org.apache.commons.io.FileUtils;
+import org.rocksdb.Options;
+import org.rocksdb.RocksDB;
+import org.rocksdb.RocksDBException;
+import org.rocksdb.RocksIterator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.charset.Charset;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * A page store implementation which utilizes rocksDB to persist the data.
+ */
+@NotThreadSafe
+public class RocksPageStore implements PageStore {
+  private static final Logger LOG = LoggerFactory.getLogger(RocksPageStore.class);
+  public static final int KEY_LEN = Long.BYTES * 2;
```

Comment:
What is the format of the key?

Reply:
It is `PageIndex(long) + FileIdentifier(String)`. This constant here is only used for estimation of the overhead.